tests/functional/mesh/mesh.go

```diff
@@ -132,6 +133,42 @@ func (n *Node) WebsocketDial(address string, cost float64) error {
 	return err
 }
 
+// Ping a node and wait for a response
+func (n *Node) Ping(node string) (map[string]interface{}, error) {
+	pc, err := n.NetceptorInstance.ListenPacket("")
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		_ = pc.Close()
+	}()
+	startTime := time.Now()
+	replyChan := make(chan net.Addr)
+	go func() {
+		buf := make([]byte, 8)
+		_, addr, err := pc.ReadFrom(buf)
+		if err == nil {
+			replyChan <- addr
+		}
+	}()
+	_, err = pc.WriteTo([]byte{}, netceptor.NewAddr(node, "ping"))
```

Comment:
We might want to call WriteTo in a loop here, because PacketConn packets aren't guaranteed to arrive